circle.yml

```diff
@@ -1,4 +1,30 @@
-general:
-  branches:
-    only:
-      - /circle-.*/
+machine:
+  xcode:
+    version: 7.3
```

Comment:
Apple just released Xcode 8.2 - we should probably test this against Xcode 8.1 at the very least. Circle normally has some instability in their containers when they upgrade to latest xcode so thus why I'm not saying to use 8.2... yet. I don't think we're planning on supporting anything earlier than iOS 9 with the iOS SDK, or OS X 10.11, so Xcode 8.1 should work just fine.